harness/determined/exec/harness.py

```diff
@@ -3,11 +3,13 @@
 import faulthandler
 import logging
 import sys
-from typing import Iterator
+from typing import Iterator, Optional, Dict
 
 import determined as det
-from determined import core, horovod, load
+from determined import core, horovod, load, pytorch
 from determined.common.api import analytics, certs
+from determined.pytorch import PyTorchTrial, PyTorchTrialController, PyTorchTrialContext
```

Comment:
[google style guide for python imports](https://google.github.io/styleguide/pyguide.html#22-imports)